### PR TITLE
Fix #8588: Guest list scrolling breaks above ~2000 guests

### LINKED ISSUE #8588: guest list
-->
**OS:** Windows 10
**Version:** 0.2.1
**Commit/Build:** 8ac731e

Hello! 
I seem to be encountering a problem when viewing the guests in the Guest Window. When I scroll down the lowest I can go, I cannot see all of my guests. Even when I expand the window, the lowest it will go is to the "R" names. I took a screenshot to show you. I get the same problem when I select individual guest's thoughts as well. 
Any thoughts on what I can do to fix this? 

Thanks so much!
  Fill in the placeholders below. Delete any headings and placeholders that you do not fill in.
volcania 2019-01-14 04-36-04

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. 
2. 

**Dump file**

**Screenshots / Video:**

**Save game:**

### PR DESCRIPTION
**Pull request by tomlankhorst**
Fix #8588

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Sounds like an unfortunate side-effect of #6522. This has been in for a while…!

**Comment by Gymnasiast** [comment]
I have tried this myself, even lowering the guests-per-page to 2000, but haven't created a PR because it wouldn't solve the problem. Perhaps you have, but I'd like to test that.

**Comment by tomlankhorst** [comment]
@Gymnasiast I've revised to scroll window height limit too, which was incorrectly set to 10*lines instead of 12*lines.

File: src/openrct2-ui/windows/GuestList.cpp [hunk 1]
```diff
@@ -490,7 +492,7 @@ static void window_guest_list_scrollgetsize(rct_window* w, int32_t scrollIndex,
             }
             w->var_492 = numGuests;
             y = numGuests * SCROLLABLE_ROW_HEIGHT;
-            _window_guest_list_num_pages = (int32_t)std::ceil((float)numGuests / 3173);
+            _window_guest_list_num_pages = 1 + (numGuests - 1) / GUESTS_PER_PAGE;
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/GuestList.cpp]:
```suggestion
            _window_guest_list_num_pages = ((numGuests - 1) / GUESTS_PER_PAGE) + 1;
```

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/GuestList.cpp]: Personally, I think the original is clearer. There is, after all, always one page.

Comment (Gymnasiast) [reply src/openrct2-ui/windows/GuestList.cpp]: Meh, not sure if I agree, but it's too minor to argue about. I'll merge it as-is.

**Comment by tomlankhorst** [comment]
@Gymnasiast Thanks for reviewing. I'll take care of the messages ;).

File: src/openrct2-ui/windows/GuestList.cpp (hunk 1 above)
Comment (tomlankhorst) [reply src/openrct2-ui/windows/GuestList.cpp]: I would agree with reversing the order but I don't think that the redundant parentheses add much to clarity, or was there a different motivation?

**Pull request merged**